webapp/src/ts/actions/tasks.ts

```diff
@@ -0,0 +1,81 @@
+import {createAction, Store} from '@ngrx/store';
+import { createSingleValueAction } from './actionUtils';
+
+export const Actions = {
+  setSelectedTask: createSingleValueAction('SET_SELECTED_TASK', 'selected'),
```

Comment:
I'm not sure all these actions are needed. The old reducer only had two actions.